example_dags/example_transform.py

```diff
@@ -0,0 +1,107 @@
+from datetime import datetime
+
+import pandas as pd
+from airflow import DAG
+
+from astro import sql as aql
+from astro.files import File
+from astro.sql.table import Table
+
+START_DATE = datetime(2000, 1, 1)
+LAST_ONE_DF = pd.DataFrame(data={"Title": ["Random movie"], "Rating": [121]})
+
+
+# [START transform_example_1]  skipcq: PY-W0069
+@aql.transform()
+def top_five_animations(input_table: Table):
+    return """
+        SELECT *
+        FROM {{input_table}}
+        WHERE Genre1=='Animation'
+        ORDER BY Rating desc
+        LIMIT 5;
+    """
+
+
+# [END transform_example_1]  skipcq: PY-W0069
+
+
+# [START transform_example_2]  skipcq: PY-W0069
+@aql.transform()
+def last_five_animations(input_table: Table):
+    return """
+        SELECT *
+        FROM {{input_table}}
+        WHERE Genre1=='Animation'
+        ORDER BY Rating asc
+        LIMIT 5;
+    """
+
+
+# [END transform_example_2]  skipcq: PY-W0069
+
+
+# [START transform_example_3]  skipcq: PY-W0069
+@aql.transform
+def union_top_and_last(first_table: Table, second_table: Table):
+    """Union `first_table` and `second_table` tables to create a simple dataset."""
+    return """
+            SELECT Title, Rating from {{first_table}}
+            UNION
+            SELECT Title, Rating from {{second_table}};
+            """
+
+
+# [END transform_example_3]  skipcq: PY-W0069
+
+
+# [START transform_example_4]  skipcq: PY-W0069
+@aql.transform
+def union_table_and_dataframe(input_table: Table, input_dataframe: pd.DataFrame):
+    """Union `union_table` table and `input_dataframe` dataframe to create a simple dataset."""
+    return """
+            SELECT Title, Rating from {{input_table}}
+            UNION
+            SELECT Title, Rating from {{input_dataframe}};
+            """
+
+
+# [END transform_example_4]  skipcq: PY-W0069
+
+
+with DAG(
+    "example_transform",
+    schedule_interval=None,
+    start_date=START_DATE,
+    catchup=False,
+) as dag:
+
+    imdb_movies = aql.load_file(
+        input_file=File(
+            path="https://raw.githubusercontent.com/astronomer/astro-sdk/main/tests/data/imdb.csv"
```

Comment:
@sunank200 I think referring `https://raw.githubusercontent.com/astronomer/astro-sdk/main/tests/data/imdb.csv` was giving some issues in the past. Would it be possible to either refer to a local dataset or something on storages? I think it will help our CI avoid running into flaky test issues.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/628/commits/1b59d98d45e747409368ed0f01c235ab6841dc70)